lib/ueberauth/strategy/cognito/jwt_utilities.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Ueberauth.Strategy.Cognito.JwtUtilities do
+  @moduledoc """
+  Utilities for working with JSON Web Tokens
+  """
+
+  @doc "Verifies that a JWT is valid: the signature is correct,
+  and the audience is the AWS client_id"
+  def verify(jwt, jwks, client_id, aws_region, user_pool_id) do
+    individual_jwks = Enum.map(jwks["keys"], &JOSE.JWK.from(&1))
```

Comment:
Seems like this could also be part of `verified_claims`?

Reply:
Done.